tasks/test_helper/builders/data_handling/data_pre_process.py

```diff
@@ -0,0 +1,58 @@
+import random as rand
+
+import h5py
+import numpy as np
+import scipy.sparse
+from scipy.sparse import csc_array, random
+
+
+def simulate_csc_h5_data(
+        file_path: str = None,
+        filename: str = 'simulated_csc_h5_data.h5'
+        ) -> (scipy.sparse, tuple[int, int]):
+    """
+    ### Function that simulates h5-file and corresponding matrix for comparison and testing of functions.
+    ---
+    #### Args:
+        - file_path (str): Path to folder where h5-file will be stored. Default None
+        - filename (str): Name of h5-file. Default 'simulated_csc_h5_data.h5'
+    ---
+    #### Returns:
+        - Transpose of sparse matrix
```

Comment:
Why transpose?

Reply:
Because it returns a CSC-matrix, and scanpy automatically makes the CSC-matrix from the h5-file in the format of cells as rows and genes as columns. So I made it return the transpose so that they appear in the same format and we more easily can compare them.